cmd/juju/cloud/regions_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package cloud_test
+
+import (
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/cmd/juju/cloud"
+	_ "github.com/juju/juju/provider/all"
+	"github.com/juju/juju/testing"
+)
+
+type regionsSuite struct {
+	testing.FakeJujuXDGDataHomeSuite
+}
+
+var _ = gc.Suite(&regionsSuite{})
+
+func (s *regionsSuite) TestListRegionsInvalidCloud(c *gc.C) {
+	_, err := testing.RunCommand(c, cloud.NewListRegionsCommand(), "invalid")
+	c.Assert(err, gc.ErrorMatches, "cloud invalid not found")
+}
+
+func (s *regionsSuite) TestListRegions(c *gc.C) {
+	ctx, err := testing.RunCommand(c, cloud.NewListRegionsCommand(), "aws")
+	c.Assert(err, jc.ErrorIsNil)
+	out := testing.Stdout(ctx)
+	c.Assert(out, jc.DeepEquals, `
+Region
```

Comment:
I know that "Region" was asked for here, but it looks pointless. We asked for regions, and we are writing out exactly one thing.